src/frontend/screens/KillPasscode.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import { defineMessages, FormattedMessage } from "react-intl";
+import { StyleSheet, View } from "react-native";
+
+import MaterialIcon from "react-native-vector-icons/MaterialIcons";
+import { KILL_PASSCODE } from "../constants";
+import { LIGHT_GREY } from "../lib/styles";
+import Text from "../sharedComponents/Text";
+import { NativeNavigationComponent } from "../sharedTypes";
+import { ScrollView, TouchableOpacity } from "react-native-gesture-handler";
+
+const m = defineMessages({
+  title: {
+    id: "screens.KillPasscode.title",
+    defaultMessage: "Kill Passcode",
+  },
+  toggleMessage: {
+    id: "screens.KillPasscode.toggleMessage",
+    defaultMessage: "Use Kill Passcode",
+  },
+  instructions: {
+    id: "screens.KillPasscode.instructions",
+    defaultMessage: "Enter the code above to hide sensitive data in Mapeo",
+  },
+});
+
+export const KillPasscode: NativeNavigationComponent<"KillPasscode"> = () => {
+  const [killPasscodeEnabled, setKillPasscodeEnabled] = React.useState(false);
+
+  return (
+    <ScrollView style={styles.container}>
+      <Text style={styles.title}>
+        <FormattedMessage {...m.title} />
+      </Text>
```

Comment:
Do we want duplicate titles - e.g. in the header and in the main view?

Reply:
This was in the design that Sabella Built, but It made more sense when we had the paper key title as well. Ill take it out for now

Reply:
This was in Sabella's Design...Im not sure why this was the case. Ill ask Sabella